internal/migrations/20210713123129_populate_infra_env.go

```diff
@@ -31,7 +31,7 @@ func populateInfraEnv() *gormigrate.Migration {
 				return err
 			}
 
-			dbClusters, err := getClustersFromDB(tx)
+			dbClusters, err := common.GetClustersFromDBWhere(tx, common.UseEagerLoading, common.IncludeDeletedRecords)
```

Comment:
Why is `common.UseEagerLoading` needed here?

Reply:
I didn't dive into the infra env test, I just replaced the function... I guess it can be changed.
For example for the multiple_networks module it's mandatory to look for the networks tables